### PR TITLE
338 PR review changes up to 333

### LINKED ISSUE #338: PR review changes up to 333
To avoid propagating changes from from downstream PRs, pool request changes for features/sync-v5 work are to be done in a seperate PR, link to PR and commit should be specified on the original PR.

Adding comments for ongoing discussions to make sure all ongoing comments are resolved:

### PR DESCRIPTION
**Pull request by andreievg**
closes #338

### CODE REVIEW

**Comment by andreievg** [comment]
before merging I need to make sure all comments in underlying issues are looked at

File: server/repository/src/db_diesel/changelog_row.rs [hunk 1]
```diff
@@ -65,12 +65,11 @@ impl<'a> ChangelogRowRepository<'a> {
...
     }
 
-    /// # Arguments:
-    /// * earliest the first cursor to be included in the returned
-    /// * limit the number of entries to be returned
     pub fn changelogs(
         &self,
+        // Earliest the first cursor to be included in the returned
         earliest: u64,
+        // Limit the number of entries to be returned
```
Comment (clemens-msupply) [inline server/repository/src/db_diesel/changelog_row.rs:72]: What is the proper way to document function arguments? I only found something as I did before with the markdown list...

It should have been 
"* `earliest`: the first cursor to be returned"
though...

File: server/repository/src/db_diesel/changelog_row.rs [hunk 2]
```diff
@@ -65,12 +65,11 @@ impl<'a> ChangelogRowRepository<'a> {
         ChangelogRowRepository { connection }
     }
 
-    /// # Arguments:
-    /// * earliest the first cursor to be included in the returned
-    /// * limit the number of entries to be returned
     pub fn changelogs(
         &self,
+        // Earliest the first cursor to be included in the returned
```
Comment (clemens-msupply) [inline server/repository/src/db_diesel/changelog_row.rs:70]:
```suggestion
        // The first cursor to be returned
```

File: server/repository/src/db_diesel/changelog_row.rs (hunk 1 above)
Comment (clemens-msupply) [inline server/repository/src/db_diesel/changelog_row.rs:72]:
```suggestion
        // The number of entries to be returned
```

**Review by clemens-msupply — APPROVED** [review]

File: server/repository/src/db_diesel/changelog_row.rs (hunk 1 above)
Comment (andreievg) [reply server/repository/src/db_diesel/changelog_row.rs]: Your documentation was better then this in terms of rust docs. There was a suggestion with a question to change logs to this, I should have looked at the [guide](https://doc.rust-lang.org/rust-by-example/meta/doc.html) before making the change. 

New version: 45c00a876a925c2c9d597f913244ab80d23bde7e

Screen Shot 2022-08-03 at 11 45 45 AM

File: server/repository/src/db_diesel/changelog_row.rs (hunk 2 above)
Comment (andreievg) [reply server/repository/src/db_diesel/changelog_row.rs]: changed in https://github.com/openmsupply/open-msupply/commit/45c00a876a925c2c9d597f913244ab80d23bde7e, i guess it's technically not the first returned cursor (we just start the query with that cursor)

File: server/repository/src/db_diesel/changelog_row.rs (hunk 1 above)
Comment (andreievg) [reply server/repository/src/db_diesel/changelog_row.rs]: https://github.com/openmsupply/open-msupply/commit/45c00a876a925c2c9d597f913244ab80d23bde7e

**Comment by andreievg** [comment]
Thanks for review, I went through the comments linked to the issue, i think all is resovled

**Pull request merged**